config/database.yml

```diff
@@ -1,31 +1,18 @@
-# SQLite version 3.x
-# gem install sqlite3
-#
-# Ensure the SQLite 3 gem is defined in your Gemfile
-# gem 'sqlite3'
-#
-default: &default
```

Comment:
Pysch changes in ruby 3.3+ disable aliases by default for [reasons] so just expanding the alias in the new file.